README.md

```diff
@@ -1750,10 +1750,10 @@ Edge Services Gateway settings.
 * `resource_id`: (optional) Internal ID used in the plugin for working with the object when `use_external_resource` is `true`.
 * `gateway`:
   * `esg_id`: `resource_id` from [ESG](README.md#cloudifynsxesg).
-  * `dgw_ip`: The default gateway ip (next hop).
-  * `vnic`: (optional) The vnic index of were the default gateway is reachable on.
-  * `mtu`: (optional) The MTU of the defautl gateway (default=1500).
-  * `admin_distance`: (optional) Admin distance of the defautl route (default=1).
+  * `dgw_ip`: The default gateway `IP` (next hop).
+  * `vnic`: (optional) The `VNIC` index of were the default gateway is reachable on.
```

Comment:
~(optional) The `VNIC` index of were the default gateway is reachable on.~
(optional) The `VNIC` index **on which** the default gateway is reachable ~on~.